edalize/nextpnr.py

```diff
@@ -0,0 +1,124 @@
+import logging
+import os.path
+
+from edalize.edatool import Edatool
+from edalize.yosys import Yosys
+from importlib import import_module
+
+logger = logging.getLogger(__name__)
+
+class Nextpnr(Edatool):
+
+    argtypes = []
+
+    @classmethod
+    def get_doc(cls, api_ver):
+        if api_ver == 0:
+            yosys_help = Yosys.get_doc(api_ver)
+            nextpnr_help = {
+                "members": [
+                    {
+                        "name": "arch",
+                        "type": "String",
+                        "desc": "Target architecture. Legal values are *xilinx*",
+                    },
+                    {
+                        "name": "output_format",
+                        "type": "String",
+                        "desc": "Output file format. Legal values are *fasm*",
```

Comment:
Do we expect any other output format? Otherwise I think we can drop this

Reply:
It seems `nextpnr-xilinx` outputs fasm as a main format - can't find anything else. I dropped this in the result.